modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/field/attachment/test/AttachmentManagerTest.java

```diff
@@ -155,6 +126,30 @@ public void testGetOrAddFileEntry() throws Exception {
 			Assert.assertNotNull(fileExtensionException);
 		}
 
+		ObjectDefinition objectDefinition = _createObjectDefinition("*");
+
+		tempFileEntry = _addTempFileEntry(
+			RandomTestUtil.randomString(), ".bmp",
+			RandomTestUtil.randomString(), ContentTypes.IMAGE_BMP,
+			objectDefinition);
+
+		folder = tempFileEntry.getFolder();
+
+		ObjectField objectField = _objectFieldLocalService.getObjectField(
+			objectDefinition.getObjectDefinitionId(), "attachment");
+
+		externalReferenceCode = RandomTestUtil.randomString();
```

Comment:
@AliciaGarciaGarcia just a question: Why we're generating another `externalReferenceCode` here?